### DISCUSSION TITLE
Small demos made with Diligent 2.5.6

### DISCUSSION THREAD

**Discussion by WangHoi** [DiligentGraphics/DiligentEngine#311] [Show and tell]
Learning and playing with Diligent's newest WebGPU backend, i've managed to made some interesting demos.

- [The Earth](https://wanghoi.github.io/virtual_earth/virtual_earth.html)
![The Earth](https://wanghoi.github.io/images/section-earth.gif)
- [Endless Ocean](https://wanghoi.github.io/fft-ocean/fft-ocean.html)
![Endless Ocean](https://wanghoi.github.io/images/section-ocean.gif)

Btw, to help spread the word about this cool engine, where can i find the `Made with Diligent ` logo, I want to add it.

**Comment by TheMostDiligent** [comment]
@WangHoi Thanks for sharing these demos - they look fantastic!
We currently don't have the logo, but I will put something together.
